### PR TITLE
Features/deployment improvements

### PR DESCRIPTION
**Pull request by ancadavid**

### CODE REVIEW

File: src/docker-compose.yml [hunk 1]
```diff
@@ -1,14 +1,12 @@
 services:
   rasa:
-    image: rasa-src-ro
+    image: ancadavid/chatbot-polly:rasa-src-ro
```
Comment (IoanaAlexandru) [inline src/docker-compose.yml:3]: Can the path be renamed? Having someone's name here is not ideal.

File: srcEN/docker-compose.yml [hunk 2]
```diff
@@ -1,12 +1,13 @@
 services:
   rasa:
-    image: rasa-src-en
+    image: ancadavid/chatbot-polly:rasa-src-en
```
Comment (IoanaAlexandru) [inline srcEN/docker-compose.yml:3]: Same comment as in the other file

File: srcEN/endpoints.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@
 # https://rasa.com/docs/rasa/custom-actions
 
 action_endpoint:
-    url: "http://localhost:5055/webhook"
+    url: "http://appen:5056/webhook"
```
Comment (IoanaAlexandru) [inline srcEN/endpoints.yml:14]: Can this be something like app-en? And the other one app-ro?

**Review by IoanaAlexandru — CHANGES_REQUESTED** [review]
Small comments around naming, but overall LGTM.

File: src/docker-compose.yml (hunk 1 above)
Comment (ancadavid) [reply src/docker-compose.yml:3]: This is my username on the docker hub and it's needed to identift the repo where the images are. As we discussed, I will create an account on the docker hub with an email associated with the app. I will use the new username on the first occasion when I have to train the models again (so I will build the images as well)

File: srcEN/docker-compose.yml (hunk 2 above)
Comment (ancadavid) [reply srcEN/docker-compose.yml:3]: previous comment

File: srcEN/endpoints.yml (hunk 3 above)
Comment (ancadavid) [reply srcEN/endpoints.yml:14]: This change can be easily made and I will do it once I change the account on the docker hub if it's okay

**Review by IoanaAlexandru — APPROVED** [review]

**Pull request merged**